Update lumi-workspaces for flash quota

JMuff22 opened the pull request:
Small change to the `lumi-workspaces-0.1.eb` to print `/flash/` quota.

- `/flash/` is a separate filesystem so the `offset` is the same as `scratch`, this may change. 
- There are some issues with `lfs quota` for `/flash/` currently (instability) so maybe merge this when they have been resolved. 
- Should a sanity check be added to only print `/flash/` quota if they are in a project with access to `/flash/`? 
- Should `lumi-workspaces-0.1.eb` be kept the same and instead create a `lumi-workspaces-0.2.eb`? Then in the next SW stack update move to using `lumi-workspaces/0.2`? 

Thanks!

klust commented:
Any update like this would indeed be a version 0.2.

However, the lumi-workspaces module will be replaced by a different one with more tools that are currently being developed to also check remaining allocations. 

In principle every project should also have a directory on /flash as there is no separate request in puhuri (and i guess the CSC management system for their users) for that, but it looks like after making the flash file system available again they did not do a full catch-up for projects created during the downtime. So maybe we should consider a test for now unless it turns out that they would still do that catch-up (which I hope they will as it creates a support nightmare also).

JMuff22 commented:
- I have reverted the original commit and created a new file for version 0.2. 
- Removed some unnecessary comments from the original CSC script and added credit for posterity. Also updated the version number in the config
- Added a simple if statement check if the `/flash/...` directory exists then print the quota for it. I have no way of testing if this works (no projects without access to flash).  

I could create a PR to add these to commit the new `package/lumi-allocations` branch but I am unsure how would be best to add these additions to that branch and/or if this branch should be deleted or not in favour of that one.

klust commented:
I will not merge this pull request and close it as we went for a different solution to offer quota information on LUMI which has been rolled out today in the lumi-tools module that is now loaded by default. (And also contains a lumi-workspaces command).
